Campus-iOS/Map/MainView/Panel.swift

```diff
@@ -94,9 +96,9 @@ enum PanelPosition: CGFloat {
     case top, middle, bottom
     var rawValue: CGFloat {
         switch self {
-        case .top: return 0.5 * screenHeight/3
-        case .middle: return 1.5 * screenHeight/3
-        case .bottom: return 3.2 * screenHeight/4
+        case .top: return (1/6) * screenHeight
```

Comment:
Did u also test these Positions on an iPhone SE (4") / iPhone 13 Pro Max? (so the extreme screen sizes)

Reply:
The positions are the same as before, I just thought (1/6) * x is more readable than 0.5 * x / 3.
However, I haven't tested them again, as I thought they should be okay.

Reply:
If possible, please test that as it could have been wrong before :)